runtime/appruntime/exported/config/config.go

```diff
@@ -69,10 +74,21 @@ type Gateway struct {
 type Service struct {
 	// Name is the name of the service
 	Name string `json:"name"`
-	// URL is the URL of the service
+	// URL is the base URL of the service (including protocol and port)
 	URL string `json:"url"`
+	// Protocol is the protocol that the service talks
+	Protocol SvcProtocol `json:"protocol"`
 }
 
+type SvcProtocol uint8
+
+const (
+	Http SvcProtocol = iota
+	ProtoJson
+	ProtoBinary
+	GRPC
```

Comment:
Should these be strings instead since it's exported as JSON? Also the values don't really make sense, should we start with just HTTP for now?